app/models/miq_ui_worker.rb

```diff
@@ -2,16 +2,6 @@ class MiqUiWorker < MiqWorker
   require_nested :Runner
 
   self.required_roles = ['user_interface']
-  self.check_for_minimal_role = false
-  self.workers = lambda do
-    if MiqServer.minimal_env?
-      # Force 1 UI worker in minimal mode, unless 'no_ui' is an option, which is
-      # done when the UI worker is debugged externally, such as in Netbeans.
```

Comment:
🤣  Netbeans

Reply:
Cool bonus... this PR removes the last mention of Netbeans now 😆